.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
whats mean that line ?

Reply:
This line and husky.sh was autogenerated by husky. Script is called during hook run. It looks like that it is used for debugging purposes or getting husky configuration.

Reply:
`.husky` should be gitignored - it's auto-generated by husky on postinstall

Reply:
It is autogenerated if it is not commited. Directory .husky is now place where you set up your hooks - it corresponds to husky.config.js in v4 version.